news/2018-07-13-cwc-2018-recap-semi-finals-finals-week-1.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Quarter Finals Recap"
+date: 2018-06-30 01:00:00 +0000
+---
+
+A double-header of news and reviews from the osu!catch World Cup 2018 this week – which three teams clinched their crucial places in Finals Week 2, and when is it all coming to a head? All the information you need is right here!
```

Comment:
`coming to a head` -> `coming to an end`?

Reply:
"Coming to an end" implies it's been dragging on and we want it to be over. "Coming to a head" implies that it is building to a climax, which is something to look forward to.